matterbridge.go

```diff
@@ -51,6 +51,15 @@ func main() {
 	cfg := config.NewConfig(rootLogger, *flagConfig)
 	cfg.BridgeValues().General.Debug = *flagDebug
 
+	// if logging to a file, ensure it is closed when the program terminates
+	defer func() {
+		if f, ok := rootLogger.Out.(*os.File); ok {
+			if err := f.Sync(); err != nil {
+				f.Close()
```

Comment:
We should always close the file not only when sync has an error.
I think we can just replace this by 

```
f.Sync()
f.Close()
```
Don't need to check the error there, it's not like we're able to fix it when it errors.
Put a `// nolint:errcheck` above the `defer func()`